test/tryCatch.js

```diff
@@ -0,0 +1,68 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+describe('tryCatch', function() {
+
+  function headX(ls) {
+    return ls[0].x;
+  }
+
+  function catcher() {
+    return 10101;
+  }
+
+  it('takes two functions and return a function', function() {
+    var mayThrow = R.tryCatch(headX, catcher);
+    eq(typeof mayThrow, 'function');
+  });
+
+  it('returns a function with the same arity as the `tryer` function', function() {
+    function a1(a) { return a; }
+    function a2(a, b) { return b; }
+    function a3(a, b, c) { return c; }
+    function a4(a, b, c, d) { return d; }
+
+    eq(R.tryCatch(a1, catcher).length, a1.length);
+    eq(R.tryCatch(a2, catcher).length, a2.length);
+    eq(R.tryCatch(a3, catcher).length, a3.length);
+    eq(R.tryCatch(a4, catcher).length, a4.length);
+  });
+
+  it('returns the value of the first function if it does not throw', function() {
+    var mayThrow = R.tryCatch(headX, catcher);
+    eq(mayThrow([{x: 10}, {x: 20}, {x: 30}]), 10);
+  });
+
+  it('returns the value of the second function if the first function throws', function() {
+    function throw10() {
+      throw new Error(10);
+    }
+
+    function eCatcher(e, ls) {
+      var n = Number(e.message);
+      return ls.length === 0 ? n : n * 1000;
```

Comment:
Oh, I get it.  Then I think the description should be updated. "'Returns the value of the second function if the first function throws," doesn't capture this.